drivers/hmis/app/graphql/types/hmis_schema/disability_group.rb

```diff
@@ -46,8 +45,11 @@ class HmisSchema::DisabilityGroup < Types::BaseObject
     field :substance_use_disorder, HmisSchema::Enums::Hud::DisabilityResponse, null: true
     field :substance_use_disorder_indefinite_and_impairs, HmisSchema::Enums::Hud::NoYesReasonsForMissingData, null: true
 
+    field :date_created, GraphQL::Types::ISO8601DateTime, null: true
+    field :date_updated, GraphQL::Types::ISO8601DateTime, null: true
+
     def disabling_condition
-      object.enrollment.disabling_condition
+      object.enrollment.disabling_condition || 99
```

Comment:
should this default to the calculated value if not available?

Reply:
I don't think so.. This is used for pre-filling recent records. That feels a bit misleading to show recent records and display a Disabling Condition that is not actually what was recorded.

Reply:
What happens if you choose a previous disability set where there is a disability that is indefinite and impairing but DisablingCondition was null?
If you'd entered them that way manually, wouldn't DisablingCondition be `1`?
It'll probably be fine since submitting with a 99 will throw up a warning.

Reply:
Putting this off to https://www.pivotaltracker.com/story/show/184338150